player/bundle/bundle.bzl

```diff
@@ -11,19 +11,19 @@ MODES = {
     DEV: DEVELOPMENT,
 }
 
-def webpack(name, dist, deps, mode_shorthand, visibility, bundle_entry = None, bundle_name = None, **kwargs):
+def webpack(name, dist, deps, mode_shorthand, visibility, bundle_name = None, bundle_entry = None, **kwargs):
     output_name = bundle_name if bundle_name != None else name
     _webpack(
         name = "%s_bundle_%s" % (name, mode_shorthand),
         outs = ["dist/%s.%s.js" % (bundle_name, mode_shorthand)],
-        args = filter_empty([
+        args = [
             "--mode %s" % MODES[mode_shorthand],
-            bundle_entry,
+            bundle_entry if bundle_entry else "./$(RULEDIR)",
```

Comment:
@KetanReddy I changed the approach a little bit from your webpack PR: #24 

Essentially, I default to the prior value, `./$(RULEDIR)`, to be compatible with existing usages. I also swapped the param positions to prevent it from breaking existing integrations that passed a `bundle_name` as an unamed param.